docs/index.rst

```diff
@@ -16,6 +16,7 @@ If you need more help with Parcels, try the `Discussions page on GitHub <https:/
    :hidden:
 
    Home <self>
+   v4 <https://docs.oceanparcels.org/en/latest/v4>
```

Comment:
v4 is now an external link (with associated symbol); not sure if we can easily change this?